AllisonCarte/Creeker
# Category Form

# Description
This PR is part of [this ticket](https://trello.com/c/mrt02H71/20-category), which should add the ability to create a new category via the `New Category` button in offCanvas menu that can be accessed from the categories list.

# Testing Instructions 
1. `Git commit` or `stash`
2. `Git fetch --all`
3. `Git checkout addCategory`
4. If the most recent version of the files is not there, do a `git pull origin addCategory`
5. Execute Creeker from inside of Visual Studio (green arrow button at the top)
6. Using the CLI, run NPM Start from inside of the Client folder 
7. Login using this email: `AMistake@email.com` and this password: `AmS0213`
8. Click the `Admin` dropdown to view the admin options (not yet restricted)
9. Click `Categories` to navigate to the category list
10. Click the FaBars icon near the top right to access the offCanvas
11. Click  `New Category` from the sidebar that pops out on the left
12. Enter a name and hit save to be redirected to the category list
13. Look over the list and make sure the new category was added
# Checklist
